src/intersection-observer-polyfill.js

```diff
@@ -273,6 +285,21 @@ export class IntersectionObserverPolyfill {
       }
     }
 
+
+    // Add a mutation observer to tick ourself
+    const ampdoc = Services.ampdoc(element);
+    if (ampdoc.win.MutationObserver && !this.mutationObserver_) {
+      this.viewport_ = Services.viewportForDoc(element);
+      this.mutationObserver_ = new ampdoc.win.MutationObserver(
+          this.handleMutationObserverNotification_.bind(this)
+      );
+      this.mutationObserver_.observe(ampdoc.getRootNode(), {
```

Comment:
@jridgewell was recommending using a different doc: https://github.com/ampproject/amphtml/pull/19034#discussion_r229133003

Reply:
Thanks for catching this! 😄